include/seccomp.h.in

```diff
@@ -65,6 +65,7 @@ enum scmp_filter_attr {
 	SCMP_FLTATR_CTL_TSYNC = 4,	/**< sync threads on filter load */
 	SCMP_FLTATR_API_TSKIP = 5,	/**< allow rules with a -1 syscall */
 	SCMP_FLTATR_CTL_LOG = 6,	/**< log not-allowed actions */
+	SCMP_FLTATR_SPEC_ALLOW = 7, /**< disable SSB mitigation */
```

Comment:
My apologies, I should have caught this sooner, but there is more bikeshedding here ... This should probably be `SCMP_FLTATR_CTL_<blah>` and not `SCMP_FLTATR_<blah>` (it's a silly convention, but it's held this long).

Maybe `SCMP_FLTATR_SSBDISABLE`?

Reply:
I feel bad about mentioning this so late, so feel free to leave this as-is, and I'll submit a naming patch later if that's easier for you.